src/indexApi.ts

```diff
@@ -16,9 +16,20 @@ import { forkChildren, setupDatabase } from 'edge-server-tools'
 import express from 'express'
 import nano, { MangoSelector } from 'nano'
 
+import { logger } from './client/util'
 import { config } from './config'
 import { setupInfos } from './couchSchema'
 
+const KEY_WORDS = [
+  'displayPrivateSeed',
+  'bitcoinKey',
+  'dashKey',
+  'litecoinKey',
+  'bitcoincashKey',
+  'ethereumKey',
+  'moneroMnemonic',
+  'ethereumMnemonic'
```

Comment:
This list isn't super-complete. In the core, we use:

'allKeys',
'displayPrivateSeed',
'displayPublicSeed',
'keys',
'otpKey',
'loginKey',
'publicWalletInfo',
'recoveryKey'

Instead of using plugin-by-plugin names like `bitcoinKey`, we could use `dataKey` and `syncKey`, since the core always stores those alongside the currency-specific stuff, for *every* possible wallet type.

Reply:
ok. i'll change but I don't want to use things as generic as 'keys' as that could pop up in other ways.